src/app/booth/book-booth/book-booth.component.html

```diff
@@ -1,4 +1,6 @@
-<button class="btn btn-primary" (click)="BookBooth()" >Book a Booth</button>
+<div class="text-center">
```

Comment:
It would be nice with some margin soo there will become a little air between the button and the Navbars